dm/syncer/data_validator.go

```diff
@@ -120,6 +120,7 @@ type DataValidator struct {
 
 	// such as table without primary key
 	unsupportedTable map[string]string
+	mode             string
```

Comment:
cfg.ValidationConfig already has this field(validator worker has this cfg too), no need to maintain this field separately.